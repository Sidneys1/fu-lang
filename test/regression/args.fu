main: u8(args: str[]) = {
    return args.length;
};