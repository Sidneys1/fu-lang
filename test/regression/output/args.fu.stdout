------------- test\regression\args.fu -------------
[2m   1 | main: u8(args: str[]) = {[0m
[2m   2 |     return args[0;1m.length[2m;[0m
[2m     |[;33m                ^^^^^^^ Warning: Narrowing when implicitly converting from a `usize_t` (64bit unsigned) to a `u8` (8bit unsigned).[0;2m (test\regression\args.fu:2:16-22)
[0;2m   3 | }; [0m
[m